src/common/game-loop.ts

```diff
@@ -35,9 +35,11 @@ export default abstract class GameLoop {
   private updateCount = 0;
   public runningFPS = 0; // our current FPS
 
+  public stepCount = 0;
+
   constructor(args?: GameLoopOpt) {
     this.ups = 1000.0 / (args?.ups ?? 30);
-    this.fps = 1000.0 / (args?.fps ?? 60);
+    this.fps = 1000.0 / (args?.fps ?? 60); //bad variable name
```

Comment:
```suggestion
    this.fps = 1000.0 / (args?.fps ?? 60);
```
gör issue eller faktiskt fixa det istället elr nåt